lib/config/httpRoutes.js

```diff
@@ -4,6 +4,7 @@ module.exports = [
   {verb: 'get', url: '/users/_me/_rights', controller: 'auth', action: 'getMyRights'},
   {verb: 'get', url: '/_login/:strategy', controller: 'auth', action: 'login'},
   {verb: 'get', url: '/_logout', controller: 'auth', action: 'logout'},
+  {verb: 'get', url: '/_authenticationStrategies', controller: 'auth', action: 'getStrategies'},
```

Comment:
This HTTP route should be changed as we have more routes to be implemented in the near future.

Given our current paradigm, I believe this particular route should be defined like that:

`{verb: 'get', url: '/strategies', controller: 'auth', action: 'getStrategies'},`

This opens the possibility to add more route while being consistent with our current API.
For instance:

`{verb: 'get', url:'/strategies/_info/:strategyName', controller: 'auth', action: 'getStrategyInfo'}`